Update SQL BBEs for V2 (#3019)
ballerina-platform/ballerina-distribution: examples/mysql-atomic-batch-execute-operation/mysql_atomic_batch_execute_operation.md

# Atomic batch execute

This BBE demonstrates how to use the MySQL client to execute a batch of DDL/DML operations with the help of a 

> `transaction` to achieve the atomic behaviour. Note that the MySQL database driver JAR should be defined in

Reviewer: ```suggestion
`transaction` to achieve the atomic behaviour. Note that the MySQL database driver JAR should be defined in
```
Let's separate section for the note in Markdown format.

the `Ballerina.toml` file as a dependency.

> For a sample configuration and more information on the underlying module, see the [MySQL module](https://docs.central.ballerina.io/ballerinax/mysql/latest/).

Reviewer: ```suggestion
For a sample configuration and more information on the underlying module, see the [`mysql` module](https://docs.central.ballerina.io/ballerinax/mysql/latest/).
```
Please check and fix all the occurrences.

The MySQL connector uses database properties from MySQL version 8.0.13 onwards. Therefore, it is
recommended to use a MySQL driver version greater than 8.0.13.

::: code mysql_atomic_batch_execute_operation.bal :::

::: out mysql_atomic_batch_execute_operation.out :::

> The following util files will initialize the test database before running the bbe and cleans it up afterwards.

Reviewer: ```suggestion
The following util files will initialize the test database before running the BBE and clean it up afterward.
```
